### ISSUE TITLE
Rust 2018 edition is now stable

### ISSUE THREAD

**Issue by jonathandturner** [kbknapp/cargo-outdated#151]
With the Rust 2018 edition release, it might need a bump to know that "editions" is now stable.
```
$ cargo outdated
error: failed to parse manifest at `/home/jonathan/Source/project/Cargo.toml`

Caused by:
  editions are unstable

Caused by:
  feature `edition` is required
```

**Comment by Mike-Neto** [comment]
https://github.com/kbknapp/cargo-outdated/pull/145 has a fix for this, waiting on the merge as well

**Comment by mati865** [comment]
Duplicate of https://github.com/kbknapp/cargo-outdated/issues/141

**Comment by Cogitri** [comment]
I guess this can be closed now.
